packages/ember-data/lib/system/model/errors.js

```diff
@@ -297,6 +284,7 @@ export default Ember.Object.extend(Ember.Enumerable, Ember.Evented, {
 
     var content = get(this, 'content').rejectBy('attribute', attribute);
     get(this, 'content').setObjects(content);
+    get(this, 'errorsByAttributeName').remove(attribute);
```

Comment:
Hmm, as far as I can see the debug builds [are already used ...](https://github.com/emberjs/data/blob/73cb9247228b86b8829fff3756278a02d39941c6/tests/index.html#L47-L49)